src/oas2/guards.ts

```diff
@@ -13,3 +13,10 @@ export const isTagObject = (maybeTagObject: unknown): maybeTagObject is Tag => {
 
   return false;
 };
+
+export const isResponseObject = (maybeResponseObject: unknown): maybeResponseObject is Response =>
+  isObject(maybeResponseObject) &&
+  ('description' in maybeResponseObject ||
+    'schema' in maybeResponseObject ||
+    'headers' in maybeResponseObject ||
+    'examples' in maybeResponseObject);
```

Comment:
Isn't this a little bit too relaxed?

- description should be a string
- headers, headers and examples should be objects

Reply:
There were no guards before - I pulled over the guard from OAS3 and tweaked for OAS2. I guess it's a matter of wether we want to do a full validation? Yes description should be a string (although null is fine as well), headers should be an object.. but then should we further validate the object properties, etc? Kind of a can of worms.